### DISCUSSION TITLE
Support when opening a streaming channel

### DISCUSSION THREAD

**Discussion by DJ-SteffEN** [ampache/ampache#3576] [Q&A]
Hello,

I am using version 5.4.0-release and have always used SSL with a certificate from Let's Encrypt. During its initial setup, the following HTTPS redirect was created:
Excerpt from /etc/apache2/sites-available/000-default.conf (within <VirtualHost *:80>)
    RewriteEngine On
    RewriteCond %{HTTPS} off
    RewriteRule (.*) https://%{HTTP_HOST}%{REQUEST_URI}

I now wanted to start this streaming channel ... which unfortunately does not succeed:
![image](https://github.com/ampache/ampache/assets/111641714/c6dd5f6c-aac0-40dc-a4e8-c1a6ac52ed28)
I click the status I get a 404 - Not found page.

The corresponding error message from Ampache logging reads:
[Runtime Error] fsockopen(): Unable to connect to 127.0.0.1:8200 (Connection refused) in file /var/www/ampache/src/Repository/Model/Channel.php(537)

Unfortunately, removing the SSL rewrite did nothing, the error message remains the same.
My PHP version is PHP 8.1.20.

Does anyone have a tip for me?

Thanks in advance
DJ SteffE.N.
https://inthemix.at

**Comment by mitchray** [comment]
I'm not sure about this error in particular, but just advising that [Channels have been removed in v6](https://github.com/ampache/ampache/wiki/Channels) due to there being many issues with it

You may want to try the [alternative method](https://github.com/ampache/ampache/wiki/Ampache-Icecast-and-Liquidsoap) which replaces the functionality of Channels
